app/code/Magento/CatalogInventory/Api/Data/InventoryInterface.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Magento\CatalogInventory\Api\Data;
+
+use Magento\CatalogInventory\Api\LocationInformationInterface;
+use Magento\CatalogInventory\Api\ConfigurationInterface;
+
+/**
+ * Inventory interface
+ *
+ * @api
+ */
+interface InventoryInterface extends LocationInformationInterface
```

Comment:
WarehouseInterface is a backend representation of inventory. Inventory is meant to be used only on frontend and warehouse is going to be used to manage inventory items. Their identifiers are connected. Advice to separate them by purpose was given by Anton Krill before I started creating interfaces, and I liked the idea of the immutable object on the frontend, so the contract is neat. In this case, third-party is not bound to implemented mutation contract if warehouse management logic is different from core implementation.